ServerCore/Areas/Deployment/DeploymentConfiguration.cs

```diff
@@ -18,7 +18,7 @@ internal static void ConfigureDatabase(IConfiguration configuration, IServiceCol
                     (options => options.UseLazyLoadingProxies()
                         .UseSqlServer(configuration.GetConnectionString("PuzzleServerSQLConnectionString")));
             }
-            else if (env.IsProduction() && (Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzlehunt" || Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzleday"))
+            else if (env.IsProduction() && (Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzlehunt" || Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzleday" || Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "puzzlesafari"))
```

Comment:
Similarly, this is vestigial

Reply:
Reverted.

---
In reply to: [295580897](https://github.com/PuzzleServer/mainpuzzleserver/pull/491#discussion_r295580897) [](ancestors = 295580897)